ACCESSIBILITY.md

```diff
@@ -0,0 +1,97 @@
+# Accessibility Policy of The Turing Way Community
+
+Within _The Turing Way_ community, we define accessibility practices as a set of community-wide, behavioral, social, and technical decisions that can be taken to ensure that all are welcome and are able to enter and participate in research, data-driven processes, communities and events despite any barriers for access.
+
+While accessibility is commonly understood with respect to disability, we – meaning _The Turing Way_ community – define accessibility as giving access to groups of people that encounter different barriers for participation. Barriers may be anything related to, including, and extending beyond the following: 
+- Age
+- Disability
+- Ethnicity and race
+- Family
+- Language (written and/or spoken)
+- Geography
+- Nationality
+- Religion, faith and belief
+- Sex and gender
+- Sexual orientation
+- Socioeconomic status
+- Technical or disciplinary background
+
+We recognise the complex and interlinking ways in which intersectionality affects identity, and thereby, access or inclusion within any community project like _The Turing Way_. So in this sense, our accessibility practices may overlap with practices that enable inclusion more broadly within any community.
+
+In developing this policy, we aim to draw from the [access-centered framework](https://accesscenteredmovement.com/what-access-centered-means/) in describing access as a constant process that changes in each space with each individual. Therefore, while this policy outlines our commitment to access-centered practices within our community in a broad sense, we also document the specific practices we use within _The Turing Way_ our Community Handbook, as well as share broader recommendations that go beyond our current practices in the Guide for Accessibility. Each resource (our Accessibility Policy - this document, Community Handbook Accessibility Guidelines, and our Accessibility Guide) play a different yet overlapping role in enabling, documenting, and sharing our evolving language around accessible and inclusive collaboration.
```

Comment:
I find this paragraph a little hard to understand.

Maybe it would help to break down the three different documents and their purpose. Is it something like,

- Accessibility policy - Our vision and the rules
- Community Handbook Accessibility Guidelines - Practical advice to build accessibility in your contributions/activity 
- Accessibility Guide - A broader discussion of accessibility, not just in _The Turing Way_

Reply:
This makes sense! Great tip @JimMadge - I'm working to break this down into a short list linked here: https://annuel2.framapad.org/p/ttw-accessibility-policy

> In developing this policy, we aim to draw from the [access-centered framework](https://accesscenteredmovement.com/what-access-centered-means/) in describing access as a constant process that changes in each space with each individual. This document is a part of a broader collection of documentation listed below:
- Accessibility policy: Our vision and guiding principles for how we practice accessibility.
- Community Handbook Accessibility Guidelines: Practical advice to build accessibility into your contributions and activity within the community
- Accessibility Guide: A broader discussion of accessibility and access-related practices that extend beyond The Turing Way